scripts/entity-generation/2-create-entity-types.ts

```diff
@@ -131,34 +135,27 @@ for (const { dir, config } of configs) {
         config,
       );
 
-      for (const issue of result.issues) {
-        console.warn(
-          `::warning title=${config.metadata.domain}->${config.metadata.module} (team ${config.metadata.team})::⚠️ ${issue}`,
-        );
-      }
+      result.issues.forEach((message) => warn(config.metadata, entityType.name, { type: 'schema', message }));
 
       intermediateResults.push({
         entityType: result.entityType,
-        domain: config.metadata.domain,
-        module: config.metadata.module,
+        metadata: config.metadata,
       });
     } catch (e) {
       // critical errors like schema not existing/object, etc., that cannot be recovered from/skipped
-      console.error(
-        `::error title=${config.metadata.domain}->${config.metadata.module} (team ${config.metadata.team})::❌ ${e}`,
-      );
+      error(config.metadata, entityType.name, { type: 'schema', message: e instanceof Error ? e.message : String(e) });
+
```

Comment:
Nothing exciting changed in this file besides restructuring how errors are managed (better, unified, (my) machine-readable format)